frontend/src/metabase/reference/databases/DatabaseDetailContainer.jsx

```diff
@@ -0,0 +1,76 @@
+/* eslint "react/prop-types": "warn" */
+import React, { Component } from 'react';
+import PropTypes from "prop-types";
+import { connect } from 'react-redux';
+
+import DatabaseSidebar from './DatabaseSidebar.jsx';
+import SidebarLayout from 'metabase/components/SidebarLayout.jsx';
+import DatabaseDetail from "metabase/reference/databases/DatabaseDetail.jsx"
+
+import * as metadataActions from 'metabase/redux/metadata';
+import * as actions from 'metabase/reference/reference';
+
+import {
+    getDatabase,
+    getDatabaseId,
+    getIsEditing
+} from '../selectors';
+
+
+const mapStateToProps = (state, props) => ({
+    database: getDatabase(state, props),
+    databaseId: getDatabaseId(state, props),
+    isEditing: getIsEditing(state, props)
+});
+
+const mapDispatchToProps = {
+    ...metadataActions,
+    ...actions
+};
+
+@connect(mapStateToProps, mapDispatchToProps)
+export default class DatabaseDetailContainer extends Component {
+    static propTypes = {
+        params: PropTypes.object.isRequired,
+        location: PropTypes.object.isRequired,
+        database: PropTypes.object.isRequired,
+        isEditing: PropTypes.bool
+    };
+
+    async fetchContainerData(){
+        await actions.rFetchDatabaseMetadata(this.props, this.props.databaseId);
```

Comment:
r is shorthand for "wrapped".

Some of the actions are "wrapped" (i.e. set loading + error statuses). I'm explicitly marking until this pattern has been resolved.

Imo, I think my plan for this refactor is:
1. Unwind the whole Reference* components (this PR)
2. Unwind the edit+display single component pattern
3. Unify the way fetches, etc are handled with the data model 

Separately, I'd like to reduce the number of things loaded, which I think can happen independently of 2 + 3.